runtime/doc/api.txt

```diff
@@ -1125,6 +1125,19 @@ nvim_open_term({buffer}, {opts})                            *nvim_open_term()*
     |nvim_chan_send()| can be called immediately to process sequences in a
     virtual terminal having the intended size.
 
+    Example: this `TermHl` command can be used to display and highlight raw
+    ANSI termcodes, so you can use Nvim as a "scrollback pager" (for terminals
+    like kitty):                                   *terminal-scrollback-pager* >lua
+        vim.api.nvim_create_user_command('TermHl', function()
+          local b = vim.api.nvim_create_buf(false, true)
+          local chan = vim.api.nvim_open_term(b, {})
+          vim.api.nvim_chan_send(chan, table.concat(
+            vim.api.nvim_buf_get_lines(0, 0, -1, false), '\n')
```

Comment:
Nit: The misalignment of the `table.concat` parentheses makes it seem as if the parentheses were unbalanced. Having the entire `nvim_chan_send` in a single line "reads better" IMO.